ros2param/test/test_cli.py

```diff
@@ -0,0 +1,718 @@
+                    # ignore launch_ros and ros2cli daemon nodes
+                    filtered_patterns=['.*ros2cli.*'],
+                    filtered_rmw_implementation=rmw_implementation
+                )
+            ) as node_command:
+                yield node_command
+        cls.launch_node_command = launch_node_command
+
+    # Set verb tests
+    @launch_testing.markers.retry_on_failure(times=3)
+    def test_set_invalid_param_existing_node(self):
+        with self.launch_node_command(
+                arguments=['set', '/param_node', 'unexistent_parameter', '1']) as node_command:
+            assert node_command.wait_for_shutdown(timeout=5)
+        assert node_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=[
+                "Setting parameter failed: ('Invalid access to undeclared parameter(s)', [])"],
+            text=node_command.output,
+            strict=True
+        )
+
+    @launch_testing.markers.retry_on_failure(times=3)
+    def test_set_valid_param_existing_node(self):
+        with self.launch_node_command(
+                arguments=['set', '/param_node', 'double_param', '3.14']) as node_command:
+            assert node_command.wait_for_shutdown(timeout=5)
+        assert node_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=['Set parameter successful'],
+            text=node_command.output,
+            strict=True
+        )
+
+    @launch_testing.markers.retry_on_failure(times=3)
+    def test_set_invalid_param_hidden_node_no_hidden_argument(self):
+        with self.launch_node_command(
+                arguments=[
+                    'set', '/_hidden_param_node', 'unexistent_parameter', '1']) as node_command:
+            assert node_command.wait_for_shutdown(timeout=5)
+        assert node_command.exit_code == 1
+        assert launch_testing.tools.expect_output(
+            expected_lines=['Node not found'],
+            text=node_command.output,
+            strict=True
+        )
+
+    @launch_testing.markers.retry_on_failure(times=3)
+    def test_set_invalid_param_hidden_node_hidden_argument(self):
+        with self.launch_node_command(
+                arguments=[
+                    'set', '/_hidden_param_node',
+                    'unexistent_parameter', '1', '--include-hidden-nodes']) as node_command:
+            assert node_command.wait_for_shutdown(timeout=5)
+        assert node_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=[
+                "Setting parameter failed: ('Invalid access to undeclared parameter(s)', [])"],
+            text=node_command.output,
+            strict=True
+        )
+
+    @launch_testing.markers.retry_on_failure(times=3)
+    def test_set_inexisting_node(self):
```

Comment:
@BMarchi nit: I think I'd normalize to `nonexistent`. `unexistent` seems to be valid but rare, and I can't find a convincing example of `inexisting` existence (no pun intended).

Reply:
Typo